packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx

```diff
@@ -240,6 +254,27 @@ const AmountSelect = ({
     }
   };
 
+  const onWrap = async () => {
+    try {
+      const tx: TransactionRequest = {
+        to:
+          currentChain.id === 1337 ? ganacheWTokenAddress : midasSdk.chainSpecificAddresses.W_TOKEN,
+        value: amount,
+        from: address,
+        nonce: await midasSdk.signer.getTransactionCount(),
+        gasLimit: utils.hexlify(100000),
```

Comment:
Not sure if we should specify the gas limit here -- @peetzweg any ideas if we should keep this?

Reply:
Haven't read the weth docs yet. But could imagine that there is a suggested gaslimit?

Reply:
I suppose the best practice is to estimate the gasLimit: https://stackoverflow.com/a/72068272/2896045 

I am not sure we do it elsewhere, but I believe that'd be good to follow.